src/components/switch/switch.style.js

```diff
@@ -172,7 +171,7 @@ const StyledSwitch = styled.div`
         `
           ${FieldHelpStyle} {
             margin-left: 78px;
-            padding-left: ${labelSpacing * theme.spacing}px;
+            padding-left: calc(${labelSpacing} * var(--spacing100));
```

Comment:
In line 166 of `switch.component.js`, there is a propType `labelSpacing`:
```
labelSpacing: PropTypes.oneOf([1, 2]),
```
So the padding-left value will have 8 or 16 px. In my opinion we should reflect it with the tokens.
```suggestion
            padding-left: var(${labelSpacing === 1 ? '--spacing100' : '--spacing200'});
```